ONSdigital/eq-survey-runner
# Eq 14 add elastic beanstalk to gitignore

**_What**_
- Added instructions to the README for deploying the app to Elastic Beanstalk.  
- Add support for internationalization with translations to Welsh included, and support for Gaelic

**_How to Test**_

1) Follow the instructions in the README and verify you can use the EB CLI tools and deploy the application
2) Once deployed, view the application and verify it is in english.
3) Adjust your browsers languages preferences and select Welsh (cy) as the preferred language
4) View the application again and verify the welsh translations are used.

**_Who can test**_

Anybody except @weapdiv-david
